### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_Databasebackup-networkshare_ColorContrast:The tooltip provided for resource group combo box has a color contrast of 2.8:1 which is less than 3:1 criteria.

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/azuredatastudio#15670]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.11;#AINotInScope;#Colorcontrast

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10
### Repro Steps:
1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and fill all the required fields and tab till next button and press enter.
7. Check Online migration radio button and tab till next button and press enter.
8. Check "My database are on a network share" radio button.
9. Tab till resource group combo box.
10. Open Color contrast analyser and check the color contrast for tooltip.

### Actual:
The tooltip provided for resource group combo box has a color contrast of 2.8:1 which is less than 3:1 criteria.

### Expected:
The tooltip should meet the minimum color contrast ratio that is should be greater or equal to 3:1.

### User Impact:
People with low vision often have difficulty in using the combo box.

**Comment by tripathiakhil429** [comment]
closing as duplicate.
